backend/src/openarchiefbeheer/conf/locale/nl/LC_MESSAGES/django.po

```diff
@@ -1428,18 +1428,17 @@ msgstr "Hij/zij heeft ook deze medebeoordelaars verwijderd: %(removed_co_reviewe
 #: openarchiefbeheer/logging/templates/logging/destruction_list_co_reviewers_added.txt:14
 #, python-format
 msgid ""
-"User %(user)s (member of group %(groups)s) has replaced all the co-reviewers "
-"of the list \"%(list_name)s\" with: %(added_co_reviewers)s."
+"User %(user)s (member of group %(groups)s) has added the following co-reviewers "
```

Comment:
The msgid and sgid_plural cannot be modified manually, only the translation (msgstr and msgstr_plural). The english text needs to be modified in the code and then when you run `makemessages` it will be picked up.